destination.go

```diff
@@ -106,18 +107,19 @@ func NewDestinationPlugin(impl Destination) cpluginv1.DestinationPlugin {
 type destinationPluginAdapter struct {
 	impl Destination
 
-	lastPosition *csync.ValueWatcher[Position]
+	lastPosition *csync.ValueWatcher[opencdc.Position]
 	openCancel   context.CancelFunc
 
 	// write is the chosen write strategy, either single records or batches
 	writeStrategy writeStrategy
 }
 
-func (a *destinationPluginAdapter) Configure(ctx context.Context, req cpluginv1.DestinationConfigureRequest) (cpluginv1.DestinationConfigureResponse, error) {
+func (a *destinationPluginAdapter) Configure(ctx context.Context, req cplugin.DestinationConfigureRequest) (cplugin.DestinationConfigureResponse, error) {
 	ctx = DestinationWithBatch{}.setBatchEnabled(ctx, false)
 
-	params := parameters(a.impl.Parameters()).toConfigParameters()
+	params := a.impl.Parameters()
 
+	// TODO should we stop doing this here? The Processor SDK does NOT do this.
```

Comment:
This is actually addressed in https://github.com/ConduitIO/conduit-connector-sdk/pull/133.